config/locales/application_states.yml

```
@@ -181,6 +191,12 @@ en:
       by: system
       description: Application choices that are unsubmitted are automatically rejected at the end of the cycle but can be carried over in the next cycle.
 
+    awaiting_provider_decision-interview:
+      name: Provider setups up an interview
+      by: provider
+      description: |
+        Providers sets up an interview with candidates.
```

We could put a bit more detail in this one, maybe something like this?
```suggestion
        Providers can set up an interview with candidates before making a decision
```